### PR TITLE
Better settings display

### PR DESCRIPTION
**Pull request by ghost**
I think the settings page would fit better with a larger display as some buttons mess up with long traductions. TBH, these settings views are a mess. They'll have to be greatly refactored.

Before:

screen shot 2015-06-01 at 21 08 00
capture d ecran 2015-06-01 a 20 25 32

After:

capture d ecran 2015-06-01 a 21 06 18
capture d ecran 2015-06-01 a 21 06 27
capture d ecran 2015-06-01 a 21 06 51
capture d ecran 2015-06-01 a 21 06 58
capture d ecran 2015-06-01 a 21 07 15
capture d ecran 2015-06-01 a 21 07 27

### CODE REVIEW

**Comment by AugierLe42e** [comment]
Crap... One cuke to solve :'(
BTW, it solves inline CSS n°2.

**Comment by jhass** [comment]
Maybe put location and gender next to each other?

**Comment by jhass** [comment]
And I don't think we need to stretch the birthday inputs to the full width.

**Comment by AugierLe42e** [comment]
> Maybe put location and gender next to each other?

Good idea.

> And I don't think we need to stretch the birthday inputs to the full width.

This id Bootstrap default. I don't think we should change such tiny details. It will be harder to maintain if we customize too much.

**Comment by jhass** [comment]
Uh, can't you just put it into a span3 or whatever half the width is?

**Comment by jhass** [comment]
Or what about http://getbootstrap.com/css/#forms-inline ?

**Comment by AugierLe42e** [comment]
Yes, I can. What I'm worried about is the reaction when resizing to very small screens.

**Comment by jhass** [comment]
So what about the inline forms?
> This only applies to forms within viewports that are at least 768px wide.

**Comment by AugierLe42e** [comment]
I don't know. I'm not sure it fits well either. As you see it, the interface is in englush but some translation in french take much more space. That can be a problem and that is the reason I've enlarged the with here.

**Comment by AugierLe42e** [comment]
You can see an exemple on that picture, the button at the bottom:



The translation fits well in english but not in french.

**Comment by AugierLe42e** [comment]
@jhass: Looks better like this, right ? Cleaner !

capture d ecran 2015-06-02 a 20 37 38
capture d ecran 2015-06-02 a 20 37 51
capture d ecran 2015-06-02 a 20 38 11
capture d ecran 2015-06-02 a 20 38 23
capture d ecran 2015-06-02 a 20 38 34

**Comment by jhass** [comment]
Yeah, only nitpick I'd have is that the Cancel button seems a bit lonely, why not stick it right to the left of the other one?

**Comment by jhass** [comment]
Also language select feels inconsistent.

**Comment by AugierLe42e** [comment]
> Yeah, only nitpick I'd have is that the Cancel button seems a bit lonely, why not stick it right to the left of the other one?

Seems reasonnable. Let's give a try.

> Also language select feels inconsistent.

What do you mean ?

**Comment by jhass** [comment]
Well, all the inputs drift to the right but that language select is stuck to the left.

**Comment by AugierLe42e** [comment]
Yeah... I don't know... It'd feel weird to have all the elements (the input and the button) on the right. The ohter elements are on the right because they have a label on the left. Here what I can propose : the input and the button on the same line, like the *Cancel* and the *Change password* button just above?

**Comment by jhass** [comment]
Mh yeah, I guess same line would help

**Comment by AugierLe42e** [comment]
That's it. I don't think I'll be able to refacor more than this :p

capture d ecran 2015-06-03 a 00 04 16
capture d ecran 2015-06-03 a 00 04 38
capture d ecran 2015-06-03 a 00 04 49
capture d ecran 2015-06-03 a 00 04 58

Question is: why is Travic so silent ?

File: app/helpers/people_helper.rb [hunk 1]
```diff
@@ -32,7 +32,8 @@ def person_link(person, opts={})
   end
 
   def person_image_tag(person, size = :thumb_small)
-    image_tag(person.profile.image_url(size), :alt => person.name, :class => 'avatar', :title => person.name, 'data-person_id' => person.id)
+    image_tag(person.profile.image_url(size), :alt => person.name, :class => 'avatar img-responsive center-block',
```
Comment (jhass) [inline app/helpers/people_helper.rb]: And as always, while touching please upgrade to Ruby 1.9 hash syntax ;)

Comment (AugierLe42e) [inline app/helpers/people_helper.rb]: Of course.

**Comment by jhass** [comment]
Alright, if you could just squash together to get rid of those meaningless commit messages (you know `git commit --amend` btw? ;) ) we're good I think.

**Comment by AugierLe42e** [comment]
Nope, I don't know `git comit --amend`. Is she sexy ? :p

**Comment by denschub** [comment]
Dude... please be a bit more serious.

**Comment by AugierLe42e** [comment]
Why so ? You never laugh ?

**Comment by jhass** [comment]
Thanks!

**Comment by svbergerem** [comment]
@AugierLe42e I think this PR broke one cuke: https://travis-ci.org/diaspora/diaspora/jobs/65163504#L2572

**Comment by AugierLe42e** [comment]
@svbergerem Indeed. Nothing crtical, hopefully. I submit a patch tonight.

**Pull request merged**